docker-run.sh

```diff
@@ -231,10 +231,26 @@ run
 set -e
 
 if [[ ${DOLI_CRON} -eq 1 ]]; then
-    echo "PATH=\$PATH:/usr/local/bin" > /etc/cron.d/dolibarr
-    echo "*/5 * * * * www-data /var/www/scripts/cron/cron_run_jobs.php ${DOLI_CRON_KEY} ${DOLI_CRON_USER} > /var/www/documents/cron_run_jobs.php.log 2>&1" >> /etc/cron.d/dolibarr
-    cron -f
-    exit 0
+  DOLI_CRON_FILE=/etc/cron.d/dolibarr
+  if [[ -z ${DOLI_CRON_USER} ]]; then
+    echo "'DOLI_CRON_USER' is undefined or empty"
+    echo "Unable to create cron task !"
+    exit 1
+  fi
+  if [[ -z ${DOLI_CRON_KEY} ]]; then
+    echo "'DOLI_CRON_KEY' is undefined or empty"
+    echo "Unable to create cron task !"
+    exit 1
+  fi
+  # Remove old file
+  if [[ -f ${DOLI_CRON_FILE} ]]; then
+    rm -rf ${DOLI_CRON_FILE}
+  fi
```

Comment:
It's not necessary because the content will be overwritten by the first echo.

Reply:
Yes, it's true! It was just to be clearer

Reply:
You don't have to delete it only if it is there, you don't have to test, you can do `rm -rf ${DOLI_CRON_FILE}` without the if.